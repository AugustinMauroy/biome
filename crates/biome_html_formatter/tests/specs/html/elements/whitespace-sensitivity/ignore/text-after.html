<span class="avatar"><img src="user-avatar.png" alt="profile photo"/><img src="user-avatar2.png" alt="profile alt"/></span>username