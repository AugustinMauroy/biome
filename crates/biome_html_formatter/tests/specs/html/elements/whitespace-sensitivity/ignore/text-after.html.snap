---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: elements/whitespace-sensitivity/ignore/text-after.html
---
# Input

```html
<span class="avatar"><img src="user-avatar.png" alt="profile photo"/><img src="user-avatar2.png" alt="profile alt"/></span>username
```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Attribute Position: Auto
Bracket same line: false
Whitespace sensitivity: strict
Indent script and style: false
-----

```html
<span class="avatar"
	><img src="user-avatar.png" alt="profile photo" />
	<img src="user-avatar2.png" alt="profile alt" /></span
>username
```

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Attribute Position: Auto
Bracket same line: false
Whitespace sensitivity: ignore
Indent script and style: false
-----

```html
<span class="avatar">
	<img src="user-avatar.png" alt="profile photo" />
	<img src="user-avatar2.png" alt="profile alt" />
</span>
username
```
